README.md

```diff
@@ -10,36 +10,72 @@ We welcome questions/comments/feedback (and pull requests).
 * IRC - [#calico](http://webchat.freenode.net?randomnick=1&channels=%23calico&uio=d4)
 * For Calico-on-Docker specific issues, please [raise issues](https://github.com/Metaswitch/calico-docker/issues/new) on Github.
 
-## Getting started
+## Getting Started
 
-The recommended way to get started with Calico for the first time is to follow the [Getting Started Guide](docs/GettingStarted.md).  You can be up and running within a matter of minutes.  The guide includes step by step instructions covering
-- setting up a 2 node Linux cluster either
-  - using Vagrant (on Virtual Box or VMware) for easy setup on your laptop or other virtual environment
-  - by manually setting up your own cluster
-- setting up Calico on the cluster
-- creating containers using Calico networking.
+To get started using [Calico Docker networking][calico-networking], we recommend running through one or more of the

```

Comment:
I think we should be leading with the demonstrations - the call to action isn't for people to actually run our tutorials, it should be to look at the demonstrations. And then be like "hey! you can run this demo too yourself on all these different platforms"